script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -123,7 +124,7 @@ that the time is just past 9 in the morning,
 [sha:d3b6033e9e4d7c76b6b1fe5fd7f842e4a7262f8f]{
 -- Page 7, Offset 24283.
 -- ついでに言うと、アルクェイドはブーツのままベッドに乗っている事しか分からない。
-and to top it all off, Arcueid is on my bed, still wearing her boots.
+and that to top it all off, Arcueid is on my bed, boots and all.
```

Comment:
not sure about this. "boots and all" is like what I'd expect if she was some character known for her boots. I can't really put it into words, but the emphasis isn't "that crazy arcueid wearing her shoes on my bed", it's "that arcueid and her crazy boots, on my bed".

Reply:
> ... all off, Arcueid hasn't even taken her boots off before getting on my bed.

?

Reply:
Sure, that works